server/lib/backyourstack/dispatcher.js

```diff
@@ -176,6 +176,9 @@ export async function dispatch(order, subscription) {
       },
     });
   } catch (err) {
+    if (process.env.NODE_ENV !== 'production') {
+      throw err;
+    }
```

Comment:
We should not have a specific behavior like that for production. Remove it or make it the default.